packages/angular/build/src/builders/unit-test/schema.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "Unit testing",
+  "description": "Unit testing options for Angular applications.",
+  "type": "object",
+  "properties": {
+    "buildTarget": {
+      "type": "string",
+      "description": "A build builder target to serve in the format of `project:target[:configuration]`. You can also pass in more than one configuration name as a comma-separated list. Example: `project:target:production,staging`.",
+      "pattern": "^[^:\\s]*:[^:\\s]*(:[^\\s]+)?$"
+    },
+    "tsConfig": {
+      "type": "string",
+      "description": "The name of the TypeScript configuration file."
+    },
+    "runner": {
+      "type": "string",
+      "description": "The name of the test runner to use for test execution.",
+      "enum": ["vitest"]
+    },
+    "runnerConfig": {
```

Comment:
Do we intend to expose the config? As currently this option is unused.

Reply:
Removed for now.